src/platforms/flutter/upload-debug.mdx

```diff
@@ -12,13 +12,13 @@ With default settings, complete stack traces are available in your Dart error, o
 
 <Note>
 
-For Flutter iOS `split-debug-info` and `obfuscate` flags are not supported. This is a known tooling limitation, as discussed in [43612](https://github.com/dart-lang/sdk/issues/43612) and [43274](https://github.com/dart-lang/sdk/issues/43274).
+For Flutter iOS and macOS `split-debug-info` and `obfuscate` flags are not supported. This is a known tooling limitation, as discussed in [43612](https://github.com/dart-lang/sdk/issues/43612) and [43274](https://github.com/dart-lang/sdk/issues/43274).
```

Comment:
is possible to use `split-debug-info` and/or `obfuscate` on Windows/Linux? are they available?

Reply:
Windows only has `obfuscate` but Linux has both `split-debug-info` and `obfuscate`

Reply:
I dont know if we support these 2 flags on desktop yet, if it requires changes on the SDK or server, it'd need to be tested, maybe lets create an issue for that.

Support `split-debug-info` and `obfuscate` for Desktop, as a description, we can tell that its never been tested, it may work ootb though

Reply:
we need to state that on the docs though, something like:

- For Flutter Desktop (Windows/macOS/Linux) `split-debug-info` and `obfuscate` flags are tested yet, follow linktonewissue

wdyt?

Reply:
Seems fair. I created https://github.com/getsentry/sentry-dart/issues/433 for it